cli/scripts/mecha-comet-gen1-r5.auto

```diff
@@ -0,0 +1,47 @@
+SDP: boot -f _flash.bin -scanlimited 0x800000
+SDPV: delay 1000
+SDPV: write -f _flash.bin -skipspl -scanterm -scanlimited 0x800000
+SDPV: jump -scanlimited 0x800000
+FB: ucmd setenv fastboot_dev mmc
+FB: ucmd setenv mmcdev ${emmc_dev}
+FB: ucmd mmc dev ${emmc_dev}
+FB: flash -scanterm -scanlimited 0x800000 bootloader _flash.bin
+FB: ucmd if env exists emmc_ack; then ; else setenv emmc_ack 0; fi;
+FB: ucmd mmc partconf ${emmc_dev} ${emmc_ack} 1 0
+FB: ucmd setenv fastboot_buffer ${loadaddr}
+FB: download -f Image 
+FB: ucmd setenv fastboot_buffer ${fdt_addr}
+FB: download -f imx8mm-mecha-comet-m-gen1.dtb
+FB: ucmd setenv fdtfile imx8mm-mecha-comet-m-gen1.dtb
+FB: ucmd printenv fdtfile 
+FB: ucmd setenv fastboot_buffer ${initrd_addr}
+FB: download -f mecha-image-mfgtool-initramfs-mecha-comet-m-gen1-20240702102938.rootfs.cpio.gz.u-boot
```

Comment:
replace all variables here that you see with arguments, we will maintain a machine configuration